src/App.tsx

```diff
@@ -1,167 +1,44 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { NavLink, Outlet, useLocation } from 'react-router-dom';
+import classNames from 'classnames';
+import { HomePage } from './components/HomePage';
+
+const NLClass = ({ isActive }: { isActive: boolean }) => {
+  return classNames('navbar-item', {
+    'has-background-grey-lighter': isActive,
+  });
+};
```

Comment:
The `NLClass` function is defined inside the `App` component. This can lead to performance issues as it will be recreated on every render. Consider moving it outside of the `App` component to optimize performance.